x-pack/legacy/plugins/monitoring/public/components/chart/get_chart_options.js

```diff
@@ -5,53 +5,66 @@
  */
 
 import { merge } from 'lodash';
-import { CHART_LINE_COLOR, CHART_TEXT_COLOR } from '../../../common/constants';
+import { CHART_TEXT_COLOR } from '../../../common/constants';
+import chrome from '../../../../../../../src/legacy/ui/public/chrome';
+import euiDarkVars from '@elastic/eui/dist/eui_theme_dark.json';
+import euiLightVars from '@elastic/eui/dist/eui_theme_light.json';
+const isDarkMode = chrome.getUiSettingsClient().get('theme:darkMode');
+
+export const euiColorForTheme = euiColor => {
```

Comment:
@cchaos What's the best way to do this in Kibana these days? I suggested this based on some older code I had, and didn't know if we have anything more utility based at this point?

Reply:
That's the correct way right now, but we should consider writing a NP plugin for this like the [EUI/Charts](https://github.com/elastic/kibana/pull/45446) one but specifically for the colors json files.

There are currently around 65 files right now pulling in these json files (and probably doing the dark theme check).

Reply:
Looking at the latest commit of my PR, we could actually just add more  methods to that since it's no longer EUI **and** Charts specific.